github/Workflow.py

```diff
@@ -160,10 +158,27 @@ def create_dispatch(
             ref = ref.name
         if inputs is NotSet:
             inputs = {}
-        status, _, _ = self._requester.requestJson(
-            "POST", f"{self.url}/dispatches", input={"ref": ref, "inputs": inputs}
-        )
-        return status == 204
+        self._requester.requestJsonAndCheck("POST", f"{self.url}/dispatches", input={"ref": ref, "inputs": inputs})
+
+    def create_dispatch(
+        self,
+        ref: github.Branch.Branch | github.Tag.Tag | github.Commit.Commit | str,
+        inputs: Opt[dict] = NotSet,
+        throw: bool = False,
+    ) -> bool:
+        """
+        Call Create Dispatch, raises or return False without details on error, depending on the "throw" parameter.
+
+        :calls: `POST /repos/{owner}/{repo}/actions/workflows/{workflow_id}/dispatches <https://docs.github.com/en/rest/reference/actions#create-a-workflow-dispatch-event>`_
+
+        """
+        try:
+            self._create_dispatch(ref, inputs)
+        except GithubException.GithubException:
+            if throw:
+                raise
+            return False
+        return True
```

Comment:
Shouldn't we still check for 202 status?

```suggestion
        return status == 204
```

Reply:
The doc says the only possible return status is 204 https://docs.github.com/en/rest/actions/workflows?apiVersion=2022-11-28#create-a-workflow-dispatch-event